ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -42,6 +42,11 @@ title: Ansible Collection Role eos_valudate_state - Preview Integration with ANT
 !!! note
     Starting from version 4.30.1F, `service_routing_protocols_model` is preset to `multi-agent` by default on EOS devices.
 
+## New Fetures and tests
+
+- AvdTestAPIHttpsSSL (No Ansible tags: Use the new `skipped_tests` variable instead)
```

Comment:
So here it looks like we will end up maintaining new tests in two places.
Can we avoid this for maintainability simplicity?

Maybe keep the test in the test categories and indicate in this paragraph that the New tests and categories are marked with the `(New)` string in the section?

Thoughts @carl-baillargeon 

```suggestion
- AvdTestAPIHttpsSSL (No Ansible tags, use the new `skipped_tests` variable instead)
```

Reply:
Yes I agree with keeping the test in test categories and create a note for the section.

I will add a "New features" only later on once we add new stuff.

Reply:
As suggested removed New features section and added note in test category section